docs/jupiterone.md

```diff
@@ -2,10 +2,11 @@
 
 ## Setup
 
-Users should configure the integration by providing an API key obtained from
-Heroku:
+Users configure the integration by providing an API key obtained from Heroku:
 <https://devcenter.heroku.com/articles/platform-api-quickstart#authentication>.
 
+**Note**: the integration currently requires a Heroku Enterprise account.
```

Comment:
"Please Note"

Just my minor opinion to ease any negative feelings for non-enterprise users, but fine either way of course.